app/models/conference.rb

```diff
@@ -82,6 +82,16 @@ def date_range_string
   end
 
   ##
+  # Check if there is any event in the program scheduled.
+  #
+  # ====Returns
+  # * +false+ ->  If there aren't schedules events
+  # * +true+ -> If there are schedules events
+  def scheduled?
```

Comment:
Because it was used only once (in the schedule view) like that:

`@conference.program.events.scheduled.any?`

So, moving it to conference, in the schedule view I can write only:

`@conference.scheduled?`

which I think is more readable.